File: src/main/java/com/homechange/api/rest/dto/offer/OffersDTO.java
package com.homechange.api.rest.dto.offer;

import java.util.List;

/**
 * Created by Nemanja on 5/21/17.
 *
 * DTO Used in offer search, it returns number of results
 * and list of OfferResponseDTOs
 */
public class OffersDTO {

	private Integer numberOfResults;
	private List<OfferResponseDTO> offers;

	public Integer getNumberOfResults() {
		return numberOfResults;
	}

	public void setNumberOfResults(Integer numberOfResults) {
		this.numberOfResults = numberOfResults;
	}

	public List<OfferResponseDTO> getOffers() {
		return offers;
	}

	public void setOffers(List<OfferResponseDTO> offers) {
		this.offers = offers;
	}
}

File: src/main/java/com/homechange/api/rest/dto/offer/OffersDTO.kt
package com.homechange.api.rest.dto.offer

/**
 * Created by Nemanja on 5/21/17.

 * DTO Used in offer search, it returns number of results
 * and list of OfferResponseDTOs
 */
class OffersDTO {

    var numberOfResults: Int? = null
    var offers: List<OfferResponseDTO>? = null
}
